src/style.css

```diff
@@ -1,3 +1,75 @@
 body {
+  font-family: Roboto, Helvetica, sans-serif;
   margin: 0;
+  padding: 0;
+  --main-color: #00acdc;
+}
+
+.header {
+  background: #fff;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-left: 50px;
+  padding-right: 50px;
+  justify-content: space-between;
+}
+
+.link-logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+
+}
+
+.nav__item {
+  line-height: 60px;
+  padding-left: 20px;
+  list-style: none;
+  text-transform: uppercase;
+  height: 60px;
+}
+
+.nav__item:first-child {
+  padding-left: 0;
```

Comment:
Replaced padding-left with margin-right. But I don't know about the height, because in the task it is specified not to use the height in the header, but to set the height of the links.